TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -1,3 +1,10 @@
+const math = require('mathjs');
+function geomean( inputArray){ 
+    let temp = math.prod(inputArray);
+    temp = math.nthRoot(temp,inputArray.length);
+    inputArray = [temp];
```

Comment:
Don't need to convert it since BenchmarkParser already does this: `curMetricValue = [curBenchVariant["metrics"][curMetric]["funcName"](curMetricValue)];`.